client/js/screens/UserProfile/components/source.js

```diff
@@ -0,0 +1,20 @@
+let source = {
```

Comment:
components dir should not be inside a screen dir. Please move it to the components dir and if it is a helper file then move it to the helpers dir. Thank you.